### PR TITLE
Make sure `LTS` packages have symlink for `pwsh` and `pwsh-lts`

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

Make sure LTS packages have symlink for pwsh and pwsh-lts

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [x] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Review by adityapatwardhan — APPROVED** [review]

File: tools/packaging/packaging.psm1 [hunk 1]
```diff
@@ -826,8 +826,21 @@ function New-UnixPackage {
         }
 
         # Destination for symlink to powershell executable
-        $Link = Get-PwshExecutablePath -IsPreview:$IsPreview -IsLTS:$LTS
+        $Link = Get-PwshExecutablePath -IsPreview:$IsPreview
         $linkSource = "/tmp/pwsh"
+        $linkInfo = [LinkInfo] @{
```
Comment (daxian-dbw) [inline tools/packaging/packaging.psm1]: I don't see this `$linkInfo` gets used anywhere. Am I missing something?

File: tools/packaging/packaging.psm1 [hunk 2]
```diff
@@ -826,8 +826,15 @@ function New-UnixPackage {
         }
```
Comment (PoshChan) [inline tools/packaging/packaging.psm1]: @TravisEz13, your last commit had 4 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CODE_OF_CONDUCT.md.https://opensource.microsoft.com/codeofconduct/ should work
```powershell
Failed to complete request to "https://opensource.microsoft.com/codeofconduct/". Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CODE_OF_CONDUCT.md.https://opensource.microsoft.com/codeofconduct/faq/ should work
```powershell
Failed to complete request to "https://opensource.microsoft.com/codeofconduct/faq/". Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://opensource.microsoft.com/codeofconduct/ should work
```powershell
Failed to complete request to "https://opensource.microsoft.com/codeofconduct/". Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://opensource.microsoft.com/codeofconduct/faq/ should work
```powershell
Failed to complete request to "https://opensource.microsoft.com/codeofconduct/faq/". Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

File: tools/ci.psm1 [hunk 3]
```diff
@@ -441,7 +441,7 @@ function Invoke-CIFinish
         [string] $NuGetKey
```
Comment (PoshChan) [inline tools/ci.psm1]: @daxian-dbw, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 5, but got 0.32999946773053.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1961
1961:             $pathologicalRatio | Should -BeGreaterThan 5
```

**Review by TravisEz13 — COMMENTED** [review]
LGTM

**Pull request merged**